src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family in case 'Roboto' doesn't load or isn't available on the user's computer. You can add a generic font like 'sans-serif' as a fallback. [Explanation](https://www.w3schools.com/cssref/pr_font_font-family.asp)